### PR TITLE
Added factorial and fibonacci series algorithms for #3

### PR DESCRIPTION
**Pull request by yedhink**

### CODE REVIEW

**Comment by abranhe** [comment]
Thank you for contributing to [@AllAlgorithms](https://github.com/AllAlgorithms/), your pr got merged! ✨

[img](https://github.com/allalgorithms/perl)

Make sure that you ⭐️ the project 🤓, and continue helping us!

Before the pull request can be merged you will need to fix some errors:

- The link to `/math` directory was always ok, and you added some weird link. You will need to remove those commits where you adedd that link to be merged.

**Original**:

original

**Yours**:

yours

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 - [Arithmetic Analysis](arithmetic-analysis)
 - [Graphs](graphs)
-- [Math](https://github.com/AllAlgorithms/shell/tree/master/math)
+- [Math](math)
```
Comment (abranhe) [inline README.md]: Thanks!!!

**Review by abranhe — APPROVED** [review]

**Comment by yedhink** [comment]
The repo has been starred and the bad link has been fixed. Hoping it clears the issues relating to the PR #9

**Comment by abranhe** [comment]
Thank you for contributing to [@AllAlgorithms](https://github.com/AllAlgorithms/), your pr got merged! ✨ Make sure that you ⭐️ the project 🤓, and continue helping us!

[img](https://github.com/allalgorithms/shell)

**Comment by yedhink** [comment]
I'm working on more contributions. Will make the PR's soon :+1:

**Pull request merged**